ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-ref.yaml

```diff
@@ -14,7 +14,14 @@ ref:
     - name: OS_CLOUD
     - name: BASE_DOMAIN
     - name: KURYR_ENABLED
-      default: "false" 
+      default: "false"
+  credentials:
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-vexxhost
+    mount_path: /var/run/cluster-secrets/openstack-vexxhost
+  - namespace: test-credentials
+    name: cluster-secrets-openstack-moc
+    mount_path: /var/run/cluster-secrets/openstack
```

Comment:
we could go even further and leave the self manage credentials for MOC unmounted, since we don't use them anyways.

Reply:
No need to make the patch smaller. 
Lets add all credentials on this patch since they will setup for other workflows. to use this step.